### PR TITLE
QPD-3617: Handel exception in delete call

### PR DESCRIPTION
**Pull request by lavkush-qu**

### CODE REVIEW

**Review by scriperdjq — APPROVED** [review]

**Review by aman-quartic — APPROVED** [review]

File: graphene_django_extras/mutation.py [hunk 1]
```diff
@@ -236,9 +236,17 @@ def delete(cls, root, info, **kwargs):
...
             cls._meta.model, info, type='delete', pk=pk
         ):
-            old_obj.delete()
-            old_obj.id = pk
-            return cls.perform_mutate(old_obj, info)
+            try:
+                old_obj.delete()
+                old_obj.id = pk
+                return cls.perform_mutate(old_obj, info)
+            except RuntimeError as e:
```
Comment (tushar-quartic) [inline graphene_django_extras/mutation.py:243]: Could you make this a generic exception...? Also won't the same be needed in the update as well?

Comment (lavkush-qu) [reply graphene_django_extras/mutation.py:243]: is anywhere we are raising generic exceptions? if yes plz let me know the case why we have to catch the generic exceptions.  need to check in for an update.

Comment (tushar-quartic) [reply graphene_django_extras/mutation.py:243]: In many places, while CRUD ops we have raised ValueErrors and Validation errors as well. So was concerned if these errors will get caught here or not.

Comment (tushar-quartic) [reply graphene_django_extras/mutation.py:243]: Also, won't this change need to be done on update and create methods as well?

**Pull request merged**